### ISSUE TITLE
Make a welcome page at site root

### ISSUE THREAD

**Issue by BillyZac** [BillyZac/ganachakra#9]
ganacakra.org is currently blank. Visiting shows an error. Make a simple static page that points people to forum.ganacakra.org, tells them what's going on, etc.

**Comment by JeffryGonzalez** [comment]
I wonder if this could be served up by the nginx install that is doing the load balancing for forums? I'll see if I can figure out what they have in the dockerfile..

**Comment by BillyZac** [comment]
Okay. That's way over my head. One alternative may be to have the DNS record for ganacakra.org point to a Firebase site. I think that would work.

**Comment by JeffryGonzalez** [comment]
Yeah! That's easier. ;) Do that. Do some design magic. Or just put a placeholder there for now.
